python_modules/dagster-pipes/dagster_pipes_tests/test_context.py

```diff
@@ -66,6 +68,22 @@ def _assert_undefined_asset_key(context, method, *args, **kwargs) -> None:
         getattr(context, method)(*args, **kwargs)
 
 
+def test_de_escape_asset_key():
+    assert de_escape_asset_key("foo") == "foo"
+    assert de_escape_asset_key("foo/bar") == "foo/bar"
+    assert de_escape_asset_key(r"foo\/bar") == "foo/bar"
```

Comment:
Add test cases per above comment for asset keys containing non-escaping backslashes